.pylintrc

```diff
@@ -0,0 +1,12 @@
+[MESSAGES CONTROL]
+# C0111: missing-docstring
+# R0903: too-few-public-methods
+# W1203: logging-fstring-interpolation
+# E0611: no-name-in-module
+# E0401: import-error
+# R0913: too-many-arguments
+# R0902: too-many-instance-attributes
+# R0801: duplicate-code
+disable=C0111, R0903, W1203, E0611, E0401, R0913, R0902, R0801
```

Comment:
suggest not ignoring R0801, It helps you write scalable code.

Reply:
OK!
Fix in 530a660.